README.md

```diff
@@ -424,6 +430,36 @@ You can connect directly to a Chrome WebSocket endpoint and include custom heade
 
 To get the WebSocket endpoint from a running Chrome instance, visit `http://127.0.0.1:9222/json/version` and look for the `webSocketDebuggerUrl` field.
 
+### Configuring default screenshot format
+
+You can set a default image format for all screenshots using the `--screenshot-format` option. The default is PNG. You can change it to JPEG or WebP if needed:
```

Comment:
I don't think we need an explicit example in the readme, let's remove it and rely on the CLI docs and `--help` examples.